sale_order_color_code_tintometric_system/models/sale_order_line.py

```diff
@@ -0,0 +1,10 @@
+# © 2024 Solvos Consultoría Informática (<http://www.solvos.es>)
+# License LGPL-3 - See http://www.gnu.org/licenses/lgpl-3.0.html
+
+from odoo import models, fields
+
+class SaleOrderLine(models.Model):
+    _inherit = "sale.order.line"
+
+    color_code_tintometric = fields.Char(string="Lot/S.Tin")
```

Comment:
Prevent copying it when duplicate functionality for Sales Orders is used:
```suggestion
    color_code_tintometric = fields.Char(string="Lot/S.Tin", copy=False)
```